test/rules/newline-per-chained-call/default/test.ts.lint

```diff
@@ -0,0 +1,78 @@
+const y: string[] = _observable
+                        .map(function(item) { return item.helloYay().another() }
+                                                                     ~~~~~~~ [ERROR]
+                                                          ~~~~~~~~ [ERROR]
+                        .operator()    
+                        .another(function(result) { return result.hello.Yay! }).wrong();
+                                                                                ~~~~~ [ERROR]
+
+SomeClass.propA.helloYay((a: number) => {
+                ~~~~~~~~ [ERROR]
+    return a + 1;
+});
+
+this.some.nested();
+          ~~~~~~ [ERROR]
```

Comment:
I don't consider one method call a chain. Either always use a minimum of 2 or make the threshold configurable